LaunchDarkly/LaunchDarkly/Util.swift

```diff
@@ -28,3 +80,28 @@ extension String {
         return true
     }
 }
+
+extension DispatchQueue {
+
+    func debouncer() -> Debouncer {
```

Comment:
Well, that's a `debouncer`, not a `debounce` method. If you don't retain it - the work item won't be executed at all. Obviously it can be misused, just like entire GCD. Happy to see a suggestion on alternative approach. Late enough I found a `throttle` in the codebase, maybe something that reuses it

Reply:
> Well, that's a debouncer, not a debounce method.

Yeah, I see that. I guess it is that it's an extension of the queue that makes me think it would be unique across usages. `DispatchQueue.main.debouncer().debounce(....)` seems like it would be kind of convenient usage, and would probably be something I'd want to attempt. 

We don't have to worry about it.